### DISCUSSION TITLE
How do i contribute?

### DISCUSSION THREAD

**Discussion by zeelsheladiya** [Klarr-Agency/Circum-icons#9] [General]
hello, the respected authority of this repository.

 I like this open-source project. I want to contribute to this project by adding more icons. I tried to find documentation about the contribution but I didn't fount. So what do I do now?  Should I fork the repo and start adding the icon to the project? please show me the way. Thank you

**Comment by Klarr-Agency** [comment]
Hi @zeelsheladiya , my name is Guillaume, I am the respected authority 😉

I’m glad that you like our project and that you want to contribute. We don’t have a contribution guide as you see…for now, but it is on our bucket list. Let me check with our designer tomorrow if we can provide some kind of guideline. I don’t want you to lose your time making some icons that do not meet our criteria. I will try to get back to you as soon as I can.

But for now may I ask you, where did you find our project and how many icons do you planned to add and which one?

**Reply by zeelsheladiya** [reply]
Hi Guillaume, I am glad to get a reply from you. thank you so much. I found this project on https://devresourc.es/. I am planning to add around 20 icons. For instance, icons for Microsoft, Google, Apple, Amazon, Country's monument icons ( Pyramid, Eiffel tower, Statue of Liberty, Taj Mahal, etc..), Earth, Torch, Screenshot, Lock orientation on/off, Scanner, Reading mode on/off, Cast, Vibrate on/off, Edit, Key, etc.

**Reply by zeelsheladiya** [reply]
any update Guillaume?

**Reply by Klarr-Agency** [reply]
Yes, I do have updates for you!

Our designer is working on a guideline and it should be available by the end of the week on our repo. Meanwhile, I'm working on the make of a CONTRIBUTING.md file with all the instruction on how to contribute. 

For now you can open an issue with the template Icon Request for each icon that you want to add. Don't forget to verify if your icon does not exist before creating your issue. You can add screenshot in reference to help us determine if we already have it.

In the list that you have provide I think we have some of them already like: scanner (barcode), reading mode on/off (read and unread) and edit (edit).

I will notify you when everything will be available.

Have a nice day!

**Reply by zeelsheladiya** [reply]
got it.

**Comment by Klarr-Agency** [comment]
Here we go! 

Our contribution [guide](https://github.com/Klarr-Agency/Circum-Icons/blob/main/CONTRIBUTING.md) is here. 🎉
You can follow the instruction from there.

Thank you for your contribution!
